ggml/src/ggml-vulkan/ggml-vulkan.cpp

```diff
@@ -13847,6 +13869,7 @@ static bool ggml_backend_vk_device_supports_op(ggml_backend_dev_t dev, const ggm
         case GGML_OP_OPT_STEP_SGD:
             return ggml_is_contiguous(op->src[0]) && op->src[0]->type == GGML_TYPE_F32;
         case GGML_OP_LOG:
+        case GGML_OP_TRI:
             return op->src[0]->type == GGML_TYPE_F32 || op->src[0]->type == GGML_TYPE_F16;
```

Comment:
We haven't really done this consistently, but to be safe, shouldn't this also be checking `op->type`? In theory the types can be different.